src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -5116,5 +5116,69 @@ static void M()
         }
 
         #endregion
+
+
+        [Fact, WorkItem(4370, "https://github.com/dotnet/roslyn/issues/4370")]
+        public void HeadingHiddenSequencePointsPickUpDocumentFromVisibleSequencePoint()
+        {
+            var source = 
+@"#line 1 ""C:\Async.cs""
```

Comment:
I was less concerned with determinism and more concerned with using a windows specific code path in a test that runs on linux :)